blog/26-01-2023 - All you need to know about Celo to Ease your Web3 Development Journey/web3-development-made-easy-with-celo

```diff
@@ -0,0 +1,14 @@
+---
+title: All you need to know about Celo to Ease your Web3 Development Journey
+description: In this tutorial, we will introduce you to the Celo blockchain and explain the key features and benefits of using it for your web3 development projects. We will cover the basics of Celo and provide an overview of its technology and ecosystem, as well as discuss the tools and resources available to help you get started. By the end of this tutorial, you will have a better understanding of Celo and how it can support your web3 development journey. Whether you are just starting out or are an experienced developer, this tutorial will provide you with the knowledge you need to get started with Celo.
+authors: 
+  - name: Abdulateef Sanni
+    tittle: Content creation (Video), Celo Sage
+    url: https://www.youtube.com/watch?v=ZpWLdkSTyyc&t=5s
```

Comment:
You can use your youtube channel profile

Reply:
instead of the link to the video right?

Reply:
or I mean where should I put my channel link?

Reply:
you can replace the link

Reply:
I have changed it to my linkedIn profile